pom.xml

```diff
@@ -198,5 +209,11 @@
                 </executions>
             </plugin>
         </plugins>
+        <extensions>
```

Comment:
It can probably be added in the parent pom I think

Reply:
It's not a problem that it's present and useless for all projects that don't use the sdk gateway?

Reply:
I don't think it is a problem to include and generalize the maven extension that just adds some maven properties based on the OS. It could be useful in other contexts than the gateway (ex: building an OS specialized distribution)

Reply:
wokay, 🚀 https://github.com/gravitee-io/gravitee-parent/pull/59